src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -829,12 +862,13 @@ private synchronized void update()
             // Route all encodings to the specified bitrate controller.
             for (RTPEncodingDesc rtpEncoding : rtpEncodings)
             {
-                logger.debug("TEMP: adding a new track projection for ssrc" +
-                        rtpEncoding.getPrimarySSRC());
                 adaptiveTrackProjectionMap.put(
                     rtpEncoding.getPrimarySSRC(),
                     adaptiveTrackProjection);
 
+                // TODO: RTX packets should never reach here (they get de-RTXed
+                // in the receiver chain). Do we still need this mapping?
+                // Perhaps we need it in order to block SR for the RTX streams?
```

Comment:
that's a really good point, in theory it should be safe to remove

Reply:
Do you know whether RTX streams have SRs?

Reply:
not in practice at least